### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by gmgreer**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -39,6 +39,25 @@ const App = (props) => {
...
+        `https://api.api-ninjas.com/v1/chucknorris?`,
+        {
+          headers: {
+            "X-Api-Key": apiKey,
+            "Content-Type": "application/json",
+          },
+        }
+      )
+      const data = await response.json()
+      console.log(data)
```
Comment (elysemontano) [inline app/javascript/components/App.js:55]: Please remove console log

File: app/javascript/components/pages/ExerciseIndex.js [hunk 2]
```diff
@@ -1,50 +1,45 @@
...
-import React, {useState} from "react";
+import React, { useState } from "react"
 import potato from "/app/assets/images/potato.jpg"
 import { NavLink, useNavigate } from "react-router-dom"
 import info from "/app/assets/images/info.png"
 
-
 const ExerciseIndex = ({ exercises, logged_in, current_user }) => {
-console.log(exercises);
+  console.log(exercises)
```
Comment (elysemontano) [inline app/javascript/components/pages/ExerciseIndex.js:7]: Please remove console log

File: app/javascript/components/pages/ExerciseShow.js [hunk 3]
```diff
@@ -1,43 +1,42 @@
...
-            <iframe className="rounded-t-2xl w-full min-h-[250px]" src={currentExercise.video} title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowFullScreen></iframe>  
-
-            <div className="p-10 flex flex-col justify-center">
-            <p>{currentExercise.description}</p>
-              <p>Category: {currentExercise.category}</p>
-              <p>Difficulty: {currentExercise.difficulty}</p>
-              
+  const { id } = useParams()
+  const currentExercise = exercises?.find((exercise) => exercise.id === +id)
+  console.log(currentExercise)
```
Comment (elysemontano) [inline app/javascript/components/pages/ExerciseShow.js:7]: Please remove console log

**Review by elysemontano — CHANGES_REQUESTED** [review]
A few console logs to remove and then you'll be good to go.

File: app/javascript/components/pages/ExerciseShow.js (hunk 3 above)
Comment (gmgreer) [reply app/javascript/components/pages/ExerciseShow.js]: Removed all console logs

**Review by frankburgs — APPROVED** [review]

**Review by frankburgs — APPROVED** [review]

**Review by frankburgs — APPROVED** [review]

**Review by frankburgs — APPROVED** [review]

**Review by elysemontano — APPROVED** [review]

**Pull request merged**